### PR TITLE
Add timestamp on Ursula

### PR DESCRIPTION
**Pull request by tuxxy**
WIP

### CODE REVIEW

File: nucypher/characters/lawful.py [hunk 1]
```diff
@@ -544,7 +545,10 @@ def __bytes__(self):
         certificate = self.rest_server_certificate()
         cert_vbytes = VariableLengthBytestring(certificate.public_bytes(Encoding.PEM))
 
-        as_bytes = bytes().join((bytes(self._interface_signature),
+        timestamp = maya.now().epoch.to_bytes(4, 'big')
```
Comment (KPrasch) [inline nucypher/characters/lawful.py]: Cool!  Looks like we're using epoch as to version ursula *each* time she is serialized?

Comment (tuxxy) [reply nucypher/characters/lawful.py]: Yes, that's what @jMyles said to do. I think this was more of a thing when we hadn't removed the DHT, though.

Comment (KPrasch) [reply nucypher/characters/lawful.py]: Yeah - its good. :+1:

**Comment by KPrasch** [comment]
Looks like there is a bug in the CircleCI configuration causing the `eth_contracts` job to run the entire test suite.  In any case, the test run produced another `ValueError`: 
https://circleci.com/gh/nucypher/nucypher/2465#queue-placeholder/containers/0

File: nucypher/characters/lawful.py (hunk 1 above)
Comment (jMyles) [reply nucypher/characters/lawful.py]: I gotta think more about this.

**Pull request merged**